augly/image/functional.py

```diff
@@ -2004,6 +2005,11 @@ def resize(
     @param height: the desired height the image should be resized to have. If
         None, the original image height will be used
 
+    @param resample: An optional resampling filter. This can be one of
+        PIL.Image.NEAREST (use nearest neighbour), PIL.Image.BILINEAR
+        (linear interpolation), PIL.Image.BICUBIC (cubic spline interpolation),
+        or PIL.Image.LANCZOS (a high-quality downsampling filter).
```

Comment:
Looks like there are actually 6 options for resample methods that `PIL.Image.resize` accepts, see [here](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.resize). Please add the 2 you're missing :)